pkg/storage/elasticsearch_dependencies.go

```diff
@@ -1,6 +1,7 @@
 package storage
 
 import (
+	"github.com/jaegertracing/jaeger-operator/pkg/account"
```

Comment:
this should be moved to the last import group

Reply:
thanks for the comment, the chance has been done